velox/experimental/cudf/exec/CudfFilterProject.h

```diff
@@ -67,18 +67,49 @@ class CudfFilterProject : public exec::Operator, public NvtxHelper {
   }
 
  private:
+  // Copied from operator FilterProject.
+  void initializeFilterProject();
+
+  /// Data for accelerator conversion.
+  struct Export {
+    const exec::ExprSet* exprs;
+    bool hasFilter;
+    const std::vector<exec::IdentityProjection>* resultProjections;
+  };
+
+  Export exprsAndProjection() const {
```

Comment:
Is this needed in `CudfFilterProject`? This was used in `FilterProject` to extract information for replacement in wave.

Also, considering how initializing operators before replacement had its problems, perhaps wave also needs a better method to extract export information. @Yuhta thoughts?